tests/InferTypesTest.elm

```diff
@@ -52,3 +53,56 @@ typeInference =
               )
             ]
         )
+
+
+typeToString : Test
+typeToString =
+    let
+        runSection ( description, tests ) =
+            describe description
+                (List.map runTest tests)
+
+        runTest ( description, input, output ) =
+            test description <|
+                \() ->
+                    Type.toString input
+                        |> Expect.equal output
+    in
+    describe "Type.toString"
+        (List.map runSection
+            [ ( "list"
+              , [ ( "empty list"
+                  , Type.List (Type.Var 0)
+                  , "List a"
+                  )
```

Comment:
Can we make a test that running it with `Type.List (Type.Var 1)` instead of `0` also results in `List a`?